packages/react-vapor/src/components/loading/components/PaginationLoading.tsx

```diff
@@ -1,41 +1,46 @@
 import * as React from 'react';
+import {ContentLoadingPlaceholder} from './ContentLoadingPlaceholder';
+import {TextLoadingPlaceholder} from './TextLoadingPlaceholder';
 
 export const PaginationLoading = () => (
     <div className="pagination-container">
         <div className="items-per-page prepended-flat-select">
             <div className="flat-select center-align">
-                <div className="text-content-placeholder mod-small mr2" />
-                <div className="flat-select-option content-placeholder mod-no-border">
+                <TextLoadingPlaceholder small className={'mr2'} />
+                <ContentLoadingPlaceholder className={'flat-select-option mod-no-border'}>
```

Comment:
Here and elsewhere in the file
```suggestion
                <ContentLoadingPlaceholder className='flat-select-option mod-no-border'>
```